extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -211,24 +211,42 @@ class QueueProcessor extends EventEmitter {
         this._redis.subscribe(channelName, err => {
             if (err) {
                 this.logger.fatal('queue processor failed to subscribe to ' +
-                                  `crr redis channel for site ${this.site}`,
+                                  `crr redis channel for location ${this.site}`,
                                   { method: 'QueueProcessor.constructor',
                                     error: err });
                 process.exit(1);
             }
-            this._redis.on('message', (channel, message) => {
-                if (channel === channelName && this._consumer[message]) {
-                    // should only validate valid functions that can be invoked
-                    // from the redis pub/sub
-                    const validCmds = ['pauseService', 'resumeService'];
-                    if (validCmds.includes(message)) {
-                        this._consumer[message](this.site);
-                    }
+            this._redis.on('message', (channel, action) => {
```

Comment:
Sorry to iterate again on this as it comes to my mind only now, but I think it would be cleaner to be more future-proof and embed the action into a JSON object like `{"action":"pauseService"}`, the reason being that there are high chances we add more commands that also require some parameters to be passed along, and it will be easier to handle if all message are JSON objects than having to check whether the message is a raw string or a JSON object.